bokeh/models/glyphs.py

```diff
@@ -349,6 +350,43 @@ class Bezier(LineGlyph):
     The {prop} values for the Bezier curves.
     """)
 
+class Block(LineGlyph, FillGlyph, HatchGlyph):
+    ''' Render rectangular regions, given a corner coordinate, width and height.
```

Comment:
I think we usually use Oxford commas, so I'd go
```suggestion
    ''' Render rectangular regions, given a corner coordinate, width, and height.
```